nav2_collision_monitor/test/sources_test.cpp

```diff
@@ -156,10 +161,47 @@ class TestNode : public nav2_util::LifecycleNode
     range_pub_->publish(std::move(msg));
   }
 
+  void publishPolygon(const rclcpp::Time & stamp)
+  {
+    polygon_pub_ = this->create_publisher<nav2_msgs::msg::PolygonsArray>(
+      POLYGON_TOPIC, rclcpp::QoS(rclcpp::KeepLast(1)).transient_local().reliable());
+
+    std::unique_ptr<nav2_msgs::msg::PolygonsArray> msg =
+      std::make_unique<nav2_msgs::msg::PolygonsArray>();
+
+    geometry_msgs::msg::PolygonStamped polygon_msg;
+
+    polygon_msg.header.frame_id = SOURCE_FRAME_ID;
+    polygon_msg.header.stamp = stamp;
+
+    geometry_msgs::msg::Point32 point;
+    point.x = 1.0;
+    point.y = 0.0;
```

Comment:
The question to make sure that everything is syched: the first point seems to asymmetrical towards to the rest 3. Is it the initial intention to make non-bar polygon, or misprint?

Reply:
good point, not intentional

Reply:
fixed